src/pages/Quote.js

```diff
@@ -0,0 +1,20 @@
+import React from 'react';
+
+const Quote = () => {
+  const quoteStyle = {
+    color: 'black',
+    fontSize: '1.8rem',
+    width: '70vw',
+    textAlign: 'center',
+  };
+
```

Comment:
Please ensure that the general layout matches these wireframes.

Your Implementation | Design
:---:|:---:
![image](https://user-images.githubusercontent.com/10332499/166108224-f264f11e-a8d2-4a30-a187-f797bd6016ca.png) | ![image](https://user-images.githubusercontent.com/10332499/166108200-d54944b7-4790-494c-9108-470fde506eac.png)